src/Writer/MSSQL.php

```diff
@@ -390,33 +382,33 @@ public function getTableInfo($tableName)
         $columns = $stmt->fetchAll(\PDO::FETCH_ASSOC);
 
         return [
-            'columns' => $columns
+            'columns' => $columns,
         ];
     }
 
-    public function testConnection()
+    public function testConnection(): void
     {
         $this->db->query('SELECT GETDATE() AS CurrentDateTime')->execute();
     }
 
-    private function handleDbError(\Exception $e, $query = '')
+    private function handleDbError(\Throwable $e, string $query = ''): UserException
     {
         $message = sprintf('DB query failed: %s', $e->getMessage());
         $exception = new UserException($message, 0, $e, ['query' => $query]);
 
         try {
             $this->db = $this->createConnection($this->dbParams);
-        } catch (\Exception $e) {
+        } catch (\Throwable $e) {
```

Comment:
Když už to tu vidím, tak by catch nemělo být nikdy IMO prázdné. Minimálně by tam měl být komentář, "je to prázdné protože...". 

Vůbec nechápu, co to dělá a proč se to připojuje, když pak v execQuery se to stejně připojuje znova....

Reply:
Na to sa este pozriem, ci by sa s tym nedalo nieco spravit :)